### ISSUE TITLE
Unblock app load when loading stories or share links.

### ISSUE THREAD

**Issue by na9da** [TerriaJS/terriajs#6501]
Currently when loading a story or share link, the app blocks until all mappable  items have been loaded. If the individual items are slow to load the app takes a while to load showing just the globe spinner. Ideally the app should render fully and show the loading state for individual workbench items.

Example: https://nationalmap.gov.au/#share=s-lbPk9xkPOTk7rDrSYDDUoPac2Nb

**Comment by nf-s** [comment]
Related to https://github.com/TerriaJS/terriajs/issues/5245

**Comment by na9da** [comment]
Closing in favour of #5245
